src/libs/core/application.h

```diff
@@ -56,6 +56,7 @@ class Application : public QObject
     Settings *settings() const;
 
     Registry::DocsetRegistry *docsetRegistry();
+    void showToolsMenu();
```

Comment:
The `Core` module should not depend on `Ui`. Currently there's a dependency on `Ui::MainWindow`, but that will go away soon. So, `Application` definitely cannot be responsible for showing a menu.